repos.yaml

```diff
@@ -310,3 +310,12 @@ repositories:
     maintainers:
       - email: ahmels@microsoft.com
         name: Ahmed ElSayed
+  - name: seldon
+    url: https://storage.googleapis.com/seldon-charts
```

Comment:
@ryandawsonuk Two things to consider...

1. You might want to put the bucket behind some DNS in case you want to move it in the future. Google Cloud buckets cannot be transferred or moved (even between projects).
2. Google Cloud is not available in China. Consider this an FYI since I find some people do not know this.